Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationBroadcaster.cs

```diff
@@ -156,17 +143,12 @@ protected void OnReceive(IncomingMessage data)
                             StateSynchronizationSceneManager.Instance.ReceiveMessage(data.Endpoint, reader);
                         }
                         break;
```

Comment:
The StateSynchronizationSceneManager could add itself as a command handler

Reply:
I'm going to avoid this for now. The "SYNC" header is used in different places by different classes. This command registration makes more sense when all headers are managed by the same class.